### ISSUE TITLE
FSE: Launch for 10% with single theme

### ISSUE THREAD

**Issue by apeatling** [Automattic/wp-calypso#35519]
With the launch of full site editing approximately on August 26, there will be a lot of feedback coming in from HEs via users. We will need to take a sprint to address fixes needed.

-----

- [x] Cleanly add FSE support to Maywood instead of Modern Business
- [x] Maywood is now the default theme when creating a new site (at least in the A/B test)
- [ ] A/B test code ready to go that also accounts for supporting Atomic transfers. (So sites don't break on upgrade)
- [ ] Verify that all blockers have been fixed
- [x] Verify that A/B testing is not blocked by a next Jetpack release (probably 7.7) (for API updates)

Tasks that are important to make part of this:
- [ ] Migrate MVP sites to FSE: https://github.com/Automattic/wp-calypso/issues/35465
- [ ] Add message when activating an older theme that uses the customizer https://github.com/Automattic/wp-calypso/issues/35646

Also see the "For Before >10% Launch" column for known bugs: https://github.com/orgs/Automattic/projects/95#column-5981922

**Comment by apeatling** [comment]
Important linked issues:

https://github.com/Automattic/wp-calypso/issues/35465

**Comment by davipontesblog** [comment]
Closing in favor of the core WordPress FSE implementation in dotCom.
